tests/test_matrix.py

```diff
@@ -298,9 +298,9 @@ def test_interpolation(
                 inter=True,
                 exp=None,
             )
-            verify(cfg, "int_missing", none=False, opt=True, missing=True, inter=True)
+            verify(cfg, "int_missing", none=False, opt=True, missing=False, inter=True)
```

Comment:
I think now that we disentangled interpolation and missing, we will be able to simplify this test.
direct test for interpolation and direct test of is missing should be sufficient.

(Might make more sense to do it after is_none is simplified as well)

Reply:
As you prefer, but I think it doesn't hurt to have a test for calling `OmegaConf.is_missing()` on an interpolation pointing to a missing value. At least it makes the intended behavior explicit. Also, the tests are checking some other flags.

Let me know if I should remove them though -- that would mean removing:
* int_missing
* int_opt_missing
* str_int_with_primitive_missing
* opt_str_int_with_primitive_missing

Reply:
> As you prefer, but I think it doesn't hurt to have a test for calling `OmegaConf.is_missing()` on an interpolation pointing to a missing value. At least it makes the intended behavior explicit. Also, the tests are checking some other flags.
> 
> Let me know if I should remove them though -- that would mean removing:
> 
> * int_missing
> * int_opt_missing
> * str_int_with_primitive_missing
> * opt_str_int_with_primitive_missing

Let's leave them in for now.